task/net/src/server.rs

```diff
@@ -561,7 +555,7 @@ where
                 self.client_waiting_to_send[socket_index] = false;
                 Ok(())
             }
-            Err(smoltcp::Error::Exhausted) => {
+            Err(smoltcp::socket::udp::SendError::BufferFull) => {
```

Comment:
(if you `use smoltcp::socket::udp` above to refer to `udp::Socket` then this becomes `udp::SendError` which'd be slightly more concise)

Reply:
Good idea, done